system/Test/Mock/MockInputOutput.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
```

Comment:
Probably a good idea to start declaring in new files so there are less surprises. 
```suggestion
declare(strict_types=1);

```